### ISSUE TITLE
npm isn't working when xlsx lib alreay installed

### ISSUE THREAD

**Issue by ryuchaehwa** [protobi/js-xlsx#145]
Hello! First of all, many thanks for this lib. It's so great.

I found an issue that if there's xlsx module installed before xlsx-style(or xlsx-styles), it doesn't install properly so I'm not available to use it. I can see it in package.json well, but in './node_modules' it's not there. only xlsx exist.
It's not a huge issue or something, but there might be some people like me seeing this error.
========================================
This relative module was not found:
* ./cptable in ./node_modules/xlsx-style/dist/cpexcel.js
========================================
btw, I'm using vscode, vue framework. If anyone finds same error msg like above, take a look at './node_modules' whether that's installed properly.

**Comment by pietersv** [comment]
@ryuchaehwa  I agree this is an important limiter for use.  I'll look to update the npm sometime this week.

**Comment by ryuchaehwa** [comment]
@pietersv Awesome! :) Thanks for this quick reply.

**Comment by bansaln** [comment]
any updates on this issue. Has this been fixed or is there a workaround to this?

**Comment by agnij** [comment]
> @ryuchaehwa 我同意这是使用的重要限制因素。我将在本周的某个时候更新npm。

Why hasn't it been updated

**Comment by ryuchaehwa** [comment]
@agnij @bansaln 
Hey guys, I don't know whether it's fixed or updated yet. I do not use this lib anymore due to our company's policy. I guess he will be able to do this when he's got time for it.  :)
